server/admin_management/api/app/services/access_control_privilege_service.py

```diff
@@ -1,18 +1,15 @@
 import logging
 from http import HTTPStatus
-from sqlalchemy.orm import Session
 from typing import List
 
 from api.app import constants as famConstants
 from api.app import schemas
-
-from api.app.services.user_service import UserService
+from api.app.repositories.access_control_privilege_repository import \
+    AccessControlPrivilegeRepository
 from api.app.services.role_service import RoleService
-from api.app.repositories.access_control_privilege_repository import (
-    AccessControlPrivilegeRepository,
-)
-
+from api.app.services.user_service import UserService
 from api.app.utils import utils
+from sqlalchemy.orm import Session
```

Comment:
maybe this import is better to stay on top, we import external resources before internal ones?

Reply:
I use VScode build-in "organize import" function.
![image](https://github.com/bcgov/nr-forests-access-management/assets/81595625/da762589-348c-48d9-9af3-a076db371592)
It is better to let VScode does automatically re-organize the file so no need to manually adjust them.